README.md

```diff
@@ -0,0 +1,123 @@
+# CMIS 1.1 compliant opencmis server
+
+## Docker
+```
+docker build -t opencmis .
+docker run -p 9000:8080 opencmis
+```
+
+## How to use it
+### CMIS 1.1
+WS (SOAP) Binding: http://localhost:9000/cmis/services11/cmis?wsdl
+AtomPub Binding: http://localhost:9000/cmis/atom11
+Browser Binding: http://localhost:9000/cmis/browser
+
+### CMIS 1.0
+WS (SOAP) Binding: http://localhost:9000/cmis/services/cmis?wsdl
+AtomPub Binding: http://localhost:9000/cmis/atom
+
+### Authentication
+Basic Authentication 
+
+Default users :
+  * user1 / cm1sp@ssword
+  * user2 / cm1sp@ssword
+  * user3 / cm1sp@ssword
+  * user4 / cm1sp@ssword
+  * user5 / cm1sp@ssword
+
+
+# Basic CMIS Docker image <!-- omit in toc -->
+
+![Docker Stars](https://img.shields.io/docker/stars/exoplatform/exo.svg) - ![Docker Pulls](https://img.shields.io/docker/pulls/exoplatform/exo.svg)
+
+- [CMIS 1.1 compliant opencmis server](#cmis-11-compliant-opencmis-server)
```

Comment:
the table of content should be moved at the top under the main title of the page

Reply:
updated